webview/src/webview_common.cpp

```diff
@@ -104,7 +104,11 @@ static int Create(lua_State* L)
     info.m_Self = dmScript::Ref(L, LUA_REGISTRYINDEX);
     info.m_L = dmScript::GetMainThread(L);
 
-    int webview_id = Platform_Create(L, &info);
+    // added by dotGears/TrungB
+    bool allowSiteAction = lua_toboolean(L, 2);
+    printf("allowSiteAction -- %s\n", allowSiteAction ? "true" : "false");
```

Comment:
Use dmLogDebug/Info/Warning/Error etc for logging.

And only log things that really need to be there for the benefit of the end user. This looks more like a debug print while testing.

Reply:
I removed